<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1508</width>
    <height>1067</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>491</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="itemIdAdd">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="shortcut">
          <string>Enter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="idList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="nameList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>520</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="regionComboBox">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="statusTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="accessibleName">
        <string notr="true"/>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="currentText">
        <string notr="true">North-America</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">North-America</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Japan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Europe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Oceania</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">China</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">中国</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="regionLabel">
       <property name="text">
        <string>Region:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dcLabel">
       <property name="text">
        <string>Data Center:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="worldLabel">
       <property name="text">
        <string>World:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="dcComboBox">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="statusTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="accessibleName">
        <string notr="true"/>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="currentText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="worldComboBox">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="statusTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="accessibleName">
        <string notr="true"/>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="currentText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1508</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuShortcuts">
    <property name="title">
     <string>Shortcuts</string>
    </property>
    <addaction name="actionAdd"/>
   </widget>
   <addaction name="menuShortcuts"/>
  </widget>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>itemIdAdd</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>nameList</tabstop>
  <tabstop>idList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>nameList</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>idList</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
